mu4/notation/internal/notationplayback.cpp

```diff
@@ -453,12 +453,26 @@ MidiData NotationPlayback::playNoteMidiData(const Ms::Note* note) const
     Ms::Instrument* instr = masterNote->part()->instrument(tick);
     channel_t channel = instr->channel(masterNote->subchannel())->channel();
 
+    Track t;
+    t.num = 0;
+    t.channels.push_back(channel);
+    midiData.tracks.push_back(t);
+
     Chunk chunk;
     chunk.beginTick = 0;
     chunk.endTick = Ms::MScore::defaultPlayDuration * 2;
-    chunk.events.insert({ chunk.beginTick, Event(channel, EventType::ME_NOTEON, pitch, 80) });
-    chunk.events.insert({ Ms::MScore::defaultPlayDuration, Event(channel, EventType::ME_NOTEOFF, pitch, 0) });
-    chunk.events.insert({ chunk.endTick, Event(channel, EventType::ME_EOT, 0, 0) });
+    auto event = Event(Event::Opcode::NoteOn);
+    event.setChannel(channel);
+    event.setNote(pitch);
+    event.setVelocityFraction(0.63); //as 80 for 127 scale
```

Comment:
This code is in 3 places. I think better to place it in one function and reuse it